app/queries/servers/post.ts

```diff
@@ -18,7 +18,7 @@ const {SERVER: {POST, POSTS_IN_CHANNEL, POSTS_IN_THREAD}} = MM_TABLES;
 
 export const prepareDeletePost = async (post: PostModel): Promise<Model[]> => {
     const preparedModels: Model[] = [post.prepareDestroyPermanently()];
-    const relations: Array<Query<Model>> = [post.drafts, post.postsInThread, post.files, post.reactions];
```

Comment:
was this me? I do think I changed it recently :S

Reply:
Not really you. I saw your change, but reverting it did not fix it.